src/flag.h

```diff
@@ -423,6 +423,16 @@ class json_flag
             return name_.translated();
         }
 
+        /** Add custom prefix to item name. */
+        std::string item_prefix() const {
+            return item_prefix_.translated();
```

Comment:
Please update the string extraction script in `lang/string_extractor/parsers/json_flag.py` to extract these new strings.